pkg/disk/controllerserver.go

```diff
@@ -1197,3 +1197,15 @@ func (cs *controllerServer) deleteUntagAutoSnapshot(snapshotID, diskID string) {
 		log.Errorf("ControllerExpandVolume:: failed to untag volumeExpandAutoSnapshot: %s", err.Error())
 	}
 }
+
+func (cs *controllerServer) ControllerGetVolume(ctx context.Context, req *csi.ControllerGetVolumeRequest,
+) (*csi.ControllerGetVolumeResponse, error) {
+	log.Infof("ControllerGetVolume is called, do nothing now")
+	return &csi.ControllerGetVolumeResponse{}, nil
```

Comment:
```suggestion
	return nil, status.Error(codes.Unimplemented, "")
```

Reply:
I modified the functions this PR involved, and the rest part's here:  https://github.com/kubernetes-sigs/alibaba-cloud-csi-driver/pull/1060